lib/hanami/logger/colorizer.rb

```diff
@@ -75,7 +75,17 @@ def app(input)
       # @since 1.2.0
       # @api private
       def severity(input)
-        colorize(input, color: colors.fetch(:severity, nil))
+        color = case Hanami::Logger.level(input)
```

Comment:
Maybe will it better to use hash of level colours instead case call?

Reply:
@davydovanton You're right. 👍 See: https://github.com/hanami/utils/pull/258/commits/d20979c7427fc956faf402ddb68f4d693cea1f46